deploy.sh

```diff
@@ -10,12 +10,11 @@ helm dep up ./helm/cas-airflow
 helm upgrade --install --timeout 900s \
   --namespace "$AIRFLOW_NAMESPACE_PREFIX-$ENVIRONMENT" \
   -f ./helm/cas-airflow/values.yaml \
+  --set-file airflow.webserver.webserverConfig=./helm/cas-airflow/webserver_config.py \
```

Comment:
This file is missing from the PR

Reply:
Ahh, it's in the gitignore, I didn't catch that. Is there a reason it's there? I wasn't awayre that we already had a config for Airflow